### PR TITLE
Add Venmo to intent filter list in AndroidManifest.xml

### LINKED ISSUE #3692: Android - Payment -Missing the dropdown for payment option, so Pay with Venmo isn't visible
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Sign-up to Expensify.cash with a US phone number and validate that account (Account A)
2. Create another account with a different US phone number (Account B) that has the Venmo app installed on their mobile device.
3. As account A, Complete the request money flow and send a payment request to account B
4. Log-in as account B and click the `pay` button
5. Verify that `Pay with Venmo` is visible as an option in the dropdown

## Expected Result:
Pay with Venmo` is visible as an option in the dropdown

## Actual Result:
Missing the dropdown for payment option, so `Pay with Venmo` isn't visible

## Workaround:
No workaround found to pay with venmo app. 

## Platform:
Where is this issue occurring?

 Web
 iOS
 Android ✔️ 
 Desktop App
 Mobile Web

**Version Number:** 1.0.72-0

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:**
Issue is not reproducible with the same account in iOS. 

image

**Expensify/Expensify Issue URL:**

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

### PR DESCRIPTION
**Pull request by luacmartins**
cc @chiragsalian 

### Fixed Issues
Fixes https://github.com/Expensify/Expensify.cash/issues/3692

### Tests
1. Sign-up to Expensify.cash with a US phone number and validate that account (Account A)
2. Create another account with a different US phone number (Account B) that has the Venmo app installed on their mobile device.
3. As account A, Complete the request money flow and send a payment request to account B (make sure the IOU report currency is USD).
4. Log-in as account B and click the pay button
5. Verify that Pay with Venmo is visible as an option in the dropdown

### QA Steps
Steps above.

### Tested On

- [ ] Web
- [ ] Mobile Web
- [ ] Desktop
- [ ] iOS
- [X] Android

#### Android Screenshots
Screenshot_1624311319
Screenshot_1624311315

### CODE REVIEW

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -7,6 +7,15 @@
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
 
+    <!-- To provide linking to services in other apps, media browser clients
+    that target Android 11+ need to add the following in their manifest -->
```
Comment (AndrewGable) [inline android/app/src/main/AndroidManifest.xml:11]: Can we just change this a bit? Maybe

```suggestion
    <!-- To provide a link to services in other apps
    that target Android 11+ we need to add the following -->
```

Comment (luacmartins) [reply android/app/src/main/AndroidManifest.xml]: Done!

**Review by AndrewGable — APPROVED** [review]

**Comment by luacmartins** [comment]
Self merging after all tests passed.

**Comment by parasharrajat** [comment]
CP staging label is needed.

**Pull request merged**